model_zoo/mnist/mnist_functional_api.py

```diff
@@ -63,7 +63,7 @@ def loss(labels, predictions):
     )
 
 
-def optimizer(lr=0.1):
```

Comment:
How about the subclass model?

Reply:
The learning rate of the subclass model is 0.01.
https://github.com/sql-machine-learning/elasticdl/blob/2225a079778313c1b8173821357c6e8d787658d4/model_zoo/mnist/mnist_subclass.py#L60